src/test/java/com/marizueva/laboratory/hw4/voidpageobject/pages/HomePage.java

```diff
@@ -0,0 +1,169 @@
+package com.marizueva.laboratory.hw4.voidpageobject.pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class HomePage extends CommonPageElements {
+
+    @FindBy(css = ".icons-benefit.icon-practise")
+    WebElement displayedBenefitImage1;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-custom']")
+    WebElement displayedBenefitImage2;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-multi']")
+    WebElement displayedBenefitImage3;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-base']")
+    WebElement displayedBenefitImage4;
```

Comment:
Images could be combined to the `List<WebElement>` collection

Reply:
fixed